cluster/examples/kubernetes/ceph/crds.yaml

```diff
@@ -924,10 +924,18 @@ spec:
                       type: object
                       nullable: true
                       properties:
-                        enabled:
+                        disabled:
                           type: boolean
                         interval:
                           type: string
+                        timeout:
+                          type: string
+                    livenessProbe:
+                      type: object
+                      nullable: true
+                      properties:
+                        disabled:
+                          type: boolean
```

Comment:
We also need the same update in the helm chart (resources.yaml) and integration tests (ceph_manifests.go)